workflow-service/src/test/java/com/redhat/parodos/workflow/execution/controller/WorkFlowControllerTest.java

```diff
@@ -231,6 +234,22 @@ public void TestGetStatusWithValidData() throws Exception {
 				.andExpect(MockMvcResultMatchers.status().isOk());
 	}
 
+	@Test
+	void TestGetStatusByProjectIdWithValidData() throws Exception {
+		UUID projectId = UUID.randomUUID();
+		Mockito.when(workFlowService.getWorkFlowStatusByProjectId(projectId))
+				.thenReturn(List.of(WorkFlowStatusResponseDTO.builder().workFlowName("test-workflow")
+						.status(ParodosWorkStatus.COMPLETED.name()).build()));
+		// when
+		this.mockMvc
+				.perform(this.getRequestWithValidCredentials("/api/v1/workflows/status").param("projectId",
+						projectId.toString()))
+				.andExpect(MockMvcResultMatchers.status().isOk())
+				.andExpect(MockMvcResultMatchers.content().contentType(MediaType.APPLICATION_JSON))
+				.andExpect(MockMvcResultMatchers.jsonPath("$", hasSize(1))).andExpect(MockMvcResultMatchers
```

Comment:
not sure if a formatter issue: can you move the last `andExpect` to a new line?

Reply:
yea. it's formatter 😅